src/containers/Payment/Payment.styles.js

```diff
@@ -95,5 +95,13 @@ export default StyleSheet.create({
 		fontSize: 20,
 		fontFamily: 'Lato-Light',
 		color: '#000'
+	},
+	icon:{
+		marginTop: 5,
+		marginLeft: 20
+	},
+	scrollView:{
```

Comment:
Brak spacji między **:** a **{** sugeruje że nie zrobiłeś przed commitem `npm run cleanup`. Chciałbym, żeby to był **standard** przed wrzuceniem każdego commita.